### ISSUE TITLE
kitchen-nodes fails serverspec test on 2012 node due to unpopulated IP address

### ISSUE THREAD

**Issue by Sam-Martin** [mwrock/kitchen-nodes#11]
## Scenario
*Vagrant:* 1.8.1  
*Test-Kitchen:* 1.4.2
*VirtualBox:* 5.0.14

*Action*: Kitchen Verify
*Expected result*: 0 examples, 0 failures

# Result
```
Failures:

         1) other node has a computername matching node fqdn
            Failure/Error:
       ::WinRM::WinRMWebService.new(
         "http://#{ip}:5985/wsman",
         :plaintext,
         user: 'vagrant',
         pass: 'vagrant',
         basic_auth_only: true
       )
            URI::InvalidURIError:
       the scheme http does not accept registry part: :5985 (or bad hostname?)
```
This is caused by a blank IP address for the Windows node in `test\integration\nodes\node?-windows-2012R2.json`.

I'll be sending a PR by with an added serverspec test that checks the IP is valid on the other node prior to trying to WinRM to it.

**Comment by mwrock** [comment]
what's the host os? windows/mac/linux?

**Comment by Sam-Martin** [comment]
Tried it on Windows 8 and Windows 10

**Comment by Sam-Martin** [comment]
- [ ]

**Comment by mwrock** [comment]
fixed vi 0e5684e21beeb4c78558b39564af953c78a0b50a
